modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/TabsGuide.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayTabs from '@clayui/tabs';
+import React, {useState} from 'react';
+
+import TokenGroup from '../components/TokenGroup';
+import TokenItem from '../components/TokenItem';
+
+const TERTIARY_TAB = (
+	<div className="align-items-center d-flex p-2">
+		<span className="inline-item inline-item-before">
+			<ClayIcon symbol="hashtag" />
+		</span>
+
+		<div className="mr-2 text-paragraph">Tertiary</div>
+
+		<div>
+			<div>
+				<h5 className="d-inline">k</h5>
+
+				<h1 className="d-inline">999.9</h1>
+
+				<h5 className="d-inline">k</h5>
+			</div>
+
+			<div className="font-weight-bold text-paragraph-sm text-right">
+				12.5%
+			</div>
+		</div>
+	</div>
+);
+
+const TABS_ACTIVE_BAR_COMPONENTS = [
+	{
+		className: 'nav-underline nav-underline-active-bar-top',
+		type: 'underline',
```

Comment:
It doesn't look like type is ever being used? Is that still needed?

Reply:
Fixed